tools/prove_wrapper

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+OUTPUT=$(mktemp)
+
+echo "Running prove with TAP output check ..."
+
+prove -I . "$@" 2>&1 | tee "$OUTPUT"
+STATUS=${PIPESTATUS[0]}
+
+if [ "$STATUS" -ne 0 ]; then
+    echo "not ok - prove failed"
+    exit "$STATUS"
+fi
+
+sed -E '
+  s/^\[[0-9]{2}:[0-9]{2}:[0-9]{2}\][[:space:]]*//
+  /^All tests successful\./,$d
+' "$OUTPUT" > "$OUTPUT.processed"
+
+UNHANDLED=$(grep -vE '^t\/.*([0-9]{2}-|)([[:alpha:]]|_|-)+\.t \.+' "$OUTPUT.processed" || true)
+
+echo "---"
```

Comment:
is that intentional? for debugging? I am not sure.

Reply:
The PR is on WIP so every strange echo or output print is indeed for debugging reasons. I'll remove everything before set the pr ready for reviews